pyhuman/app/workflows/open_office_writer.py

```diff
@@ -0,0 +1,141 @@
+import os
+import random
+import pyautogui
+from lorem.text import TextLorem
+from time import sleep
+from ..utility.base_workflow import BaseWorkflow
+
+
+WORKFLOW_NAME = 'OpenOfficeWriter'
+WORKFLOW_DESCRIPTION = 'Create documents with Apache OpenOffice Writer (Windows)'
+DEFAULT_WAIT_TIME = 2
+OPEN_OFFICE_PATH = "C:\Program Files (x86)\OpenOffice 4\program\soffice"
+
+def load():
+    return OpenOfficeWriter()
+
+class OpenOfficeWriter(BaseWorkflow):
+
+    def __init__(self, default_wait_time=DEFAULT_WAIT_TIME, open_office_path=OPEN_OFFICE_PATH):
+        super().__init__(name=WORKFLOW_NAME, description=WORKFLOW_DESCRIPTION)
+        self.default_wait_time = default_wait_time
+        self.open_office_path = open_office_path
+
+    def action(self, extra=None):
+        self._create_document()
+
+    def _create_document(self):
+        self._new_document()
+        # Type random paragrahs and sentences
+        for i in range(0, random.randint(2,10)):
+            random.choice([pyautogui.typewrite(TextLorem().paragraph()), pyautogui.typewrite(TextLorem().sentence())])
+            pyautogui.press('enter')
+        sleep(self.default_wait_time)
+        # Randomly perform actions
+        for i in range(0, random.randint(6,15)):
+            random.choice([
+                self._save_pdf,
+                self._write_sentence,
+                self._write_paragraph,
+                self._copy_paste, 
+                self._insert_comment,
+                self._find,
+                self._delete_text,
+                self._format_text])()
```

Comment:
Small formatting nitpick, but I would bump the `])()` onto the next line, since the opening `([` isn't on the same line as any of the list elements.